src/display/q1_report_generator.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+import click
+
+from config import logger
+from formula1_race_analysis.custom_exceptions import Formula1RaceAnalysisError
+from formula1_race_analysis.q1_session_analyzer import build_q1_report
+
+from .display_race_report import SortStrategy, display_race_report, filter_report, sort_report
+
+
+@click.command()
+@click.option("--data_dir", type=click.Path(exists=True, dir_okay=True, path_type=Path), required=True)
+@click.option(
+    "--order",
+    type=click.Choice(["asc", "des"], case_sensitive=False),
+    default="asc",
+    show_default=True,
+    help="['asc'] = To sort results in ascending order, ['des'] = To sort results in descending order",
+)
+@click.option("--driver", default=str)
+def generate_report(data_dir: Path, order: str, driver: str) -> None:
+    try:
+        logger.debug(f"Starting report generation. Data directory: '{data_dir}'.")
+        database = build_q1_report(Path(data_dir))
+    except Formula1RaceAnalysisError as error:
```

Comment:
![image](https://github.com/user-attachments/assets/d67894a5-0d57-4e08-82de-20b0e189271b)
I've tried to run the code not the package on exactly the same files I gave you and got